styles/google-calendar/catppuccin.user.less

```
@@ -0,0 +1,161 @@
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Agoogle-calendar
+@description Soothing pastel theme for Google Calendar
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@import "https://userstyles.catppuccin.com/lib/lib.less";
+
+// @import "https://userstyles.catppuccin.com/lib/m3.less";
+@import "https://github.com/catppuccin/userstyles/raw/refs/heads/feat/m3-lib/lib/m3.less";
+
+@-moz-document domain("calendar.google.com") {
+  // Initial load, before JS init
+  .DO879e.PC2yOb {
+    #catppuccin(@lightFlavor);
+  }
+  @media (prefers-color-scheme: dark) {
+    .DO879e.PC2yOb {
+      #catppuccin(@darkFlavor);
+    }
+  }
+
+  .DO879e:not(.PC2yOb).CcsDpe {
+    #catppuccin(@darkFlavor);
+  }
+  .DO879e:not(.PC2yOb):not(.CcsDpe) {
+    #catppuccin(@lightFlavor);
+  }
+
+  // Embed preview page
+  [data-ui-kind="1"] {
+    #catppuccin(@lightFlavor);
+  }
+
+  #catppuccin(@flavor) {
+    #lib.palette();
+    #lib.defaults();
+
+    #__m3.base();
+    #__m3.with-prefix(gm3-sys-color-);
+    #__m3.with-prefix(cal-sys-color-);
+    
+    --cal-sys-color-warning: @yellow;
+    --cal-sys-color-warning-container: fade(@yellow, @m3-tint);
+    --cal-sys-color-on-warning: @base;
+    --cal-sys-color-on-warning-container: lighten(@yellow, @m3-emphasize);
+
+    --cal-sys-color-conflict: @peach;
+    --cal-sys-color-conflict-container: fade(@peach, @m3-tint);
+    --cal-sys-color-on-conflict: @base;
+    --cal-sys-color-on-conflict-container: lighten(@peach, @m3-emphasize);
+
+    --cal-sys-color-success: @green;
+    --cal-sys-color-success-container: fade(@green, @m3-tint);
+    --cal-sys-color-on-success: @base;
+    --cal-sys-color-on-success-container: lighten(@green, @m3-emphasize);
+    
+    --cal-sys-color-night: @lavender;
+    --cal-sys-color-night-container: fade(@lavender, @m3-tint);
+    --cal-sys-color-on-night: @base;
+    --cal-sys-color-on-night-container: lighten(@lavender, @m3-emphasize);
+
+    --cal-sys-color-scrollbar: var(--cal-sys-color-outline-variant);
+    --cal-sys-color-scrollbar-hover: color-mix(in oklab, var(--cal-sys-color-outline-variant), var(--cal-sys-color-on-surface) 12%);
+    --cal-sys-color-scrollbar-active: color-mix(in oklab, var(--cal-sys-color-outline-variant), var(--cal-sys-color-on-surface) 20%);
+   
+    --cal-sys-color-now-indicator: @maroon;
```

```suggestion
    --cal-sys-color-now-indicator: @accent;
```
This should be `@accent`: accidentally regressed when refactoring.